src/test/java/com/wikia/webdriver/common/driverprovider/mobileprovider/MobileDriverProvider.java

```diff
@@ -37,8 +27,7 @@ public WebDriver getDriverInstance() {
 
     switch (platform.toUpperCase()) {
       case "ANDROID":
-        mobileDriversRegistry = new MobileDriversRegistry(platform, mobileConfig);
-        driver = getChromeDriver(platformVersion, deviceId);
+        driver = getChromeDriver();
         break;
       case "IOS":
         //@TODO
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)